api/views/statistics.py

```diff
@@ -14,16 +14,29 @@
 
 
 @extend_schema(
-    # extra parameters added to the schema
-    summary="Récapitulatif statistiques des données de ma-cantine",
+    summary="Récapitulatif statistique des données de ma-cantine",
 )
 class CanteenStatisticsView(APIView):
+    include_in_documentation = True
+    serializer_class = CanteenStatisticsSerializer
+
+    @extend_schema(
```

Comment:
j'ai un peu re-regardé pour utiliser des filterset_class, mais ca semble effectivement nécessairement rattaché à un model, et ca va filtrer le queryset en amont de la view. donc `OpenApiParameter` it is..